montecarlo/fax_waveform/TruthSorting_arrays.py

```diff
@@ -93,27 +98,27 @@
         else:
```

Comment:
Recheck this logic (in case there is a true S2 and some (photoionization) "afterpulses" (trying to change this terminology), but no S1.

Maybe consider changing input from ROOT to csv (using e.g. pandas.read_csv)